apps/api/src/database/migrations/1692870736646-seeddata.ts

```diff
@@ -85,6 +85,15 @@ export class SeedData1692870736646 implements MigrationInterface {
       ],
     );
 
+    await queryRunner.query(
+      `INSERT INTO notify_master_providers (name,provider_type,configuration) VALUES (?, ?, ?)`,
+      [
+        'SMS_KAPSYSTEM',
+        2,
+        '{"KAP_SMS_BASE_API_URL": {"lable": "KAP SMS BASE API URL","id": "KAP_SMS_BASE_API_URL","pattern": "^http://193.105.74.159/api/v3/sendsms/plain$","type": "string"},"KAP_SMS_ACCOUNT_USERNAME": {"lable": "KAP SMS ACCOUNT USERNAME","id": "KAP_SMS_ACCOUNT_USERNAME","pattern": "^[a-zA-Z0-9_-]{3,30}$","type": "string"},"KAP_SMS_ACCOUNT_PASSWORD": {"lable": "KAP_SMS_ACCOUNT_PASSWORD","id": "KAP SMS ACCOUNT PASSWORD","pattern": "^[a-zA-Z0-9_-]{3,30}$","type": "string"},"KAP_SMS_ACCOUNT_SENDER_ID": {"lable": "KAP SMS ACCOUNT SENDER ID","id": "KAP_SMS_ACCOUNT_SENDER_ID","pattern": "^[a-zA-Z]{6}$","type": "string"}}',
```

Comment:
For the seedata, the pattern for KAP_SMS_BASE_API_URL does not have to be a regex like this. We need to just have a valid URL. Current regex will only allow this address., and can cause confusion.

Reply:
Updated the pattern